### ISSUE TITLE
Users can save the same URL multiple times in their list

### ISSUE THREAD

**Issue by maximevaillancourt** [freshreader/core#100]
[Mentioned by dshannon on IndieHackers](https://www.indiehackers.com/post/feedback-for-your-project-ed3f742b5e?commentId=-MB9hZ-roqMIXQVdcaiE).

**Comment by Chhekur** [comment]
@maximevaillancourt I can work on this, but could you please elaborate more like what will be the advantage of having same URL multiple times?

**Comment by maximevaillancourt** [comment]
@Chhekur Hey! It's actually the opposite - people can currently save the same URL to their list multiple times, but the behavior I'd prefer instead is that when someone saves an URL they already have in their list, it simply moves it up at the top of their list.

(in practical terms, this means saving the URL as usual, but looking through the user's saved articles to see if the same URL already exists, and if so, deleting the old ones to only keep the most recent addition we just did)

Does that make sense?

**Comment by Chhekur** [comment]
@maximevaillancourt yeah, it make sense but before deleting the old and storing latest one, we should at least show a confirmation message to user that you already had same URL. Do you want to overwrite or something, I guess.

**Comment by maximevaillancourt** [comment]
@Chhekur I'm fine with simply showing a flash message that says something like `Looks like you already have this in your list - it's now at the top of your list and its expiry was reset` 🙂

**Comment by Chhekur** [comment]
@maximevaillancourt here's the PR [#100](https://github.com/freshreader/core/pull/110) closes this issue
